src/core/client/ui/components/AriaInfo/AriaInfo.css

```diff
@@ -0,0 +1,10 @@
+.root {
```

Comment:
Is there any document or standards doc that you can link here with justification on how this is applicable to ARIA? I tried to find some myself, but can't really find it.

Reply:
It's inspired by a proven library https://foundation.zurb.com/sites/docs/visibility.html. Most other examples are similiar, see:

-  https://www.coolfields.co.uk/2016/05/text-for-screen-readers-only-updated/
- https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/
- https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/

Reply:
However there does not seem to be a golden standard, even though most solutions look similar.